UsageDoc/CA_UsageDocument.md

```diff
@@ -92,14 +90,18 @@ CA-Tool reduces the effort in creating components in SW360 and identifying the m
    
    ![folderpic](../doc/usagedocimg/piplinepic.PNG)
 
-# Package Installation
-   *  Open CMD/PowerShell to execute below command for getting latest CA Docker image,
-         
-        `docker pull <CA_DockerImageName>`
+# Installation   
+  ### Use container image 
+
+    docker pull ghcr.io/siemens/continuous-clearing:latest
+
+ ### Nuget package
+  
+   Download the [.nupkg](https://github.com/siemens/continuous-clearing/releases) file from GitHub releases.
```

Comment:
Just keep line 100 or104 both are pointing to the same thing

Reply:
removed 104